EDA.qmd

```diff
@@ -575,7 +540,7 @@ ggplot(smaller, aes(x = carat, y = price)) +
 
 (In this section we'll use the `smaller` dataset to stay focused on the bulk of the diamonds that are smaller than 3 carats)
 
-Scatterplots become less useful as the size of your dataset grows, because points begin to overplot, and pile up into areas of uniform black (as above).
+Scatterplots become less useful as the size of your dataset grows, because points begin to overplot, and pile up into areas of uniform black, making it hard to see the trend.
```

Comment:
```suggestion
Scatterplots become less useful as the size of your dataset grows, because points begin to overplot, and pile up into areas of uniform black, making it hard to see where distribution in space.
```

That's maybe not quite right either, but it's definitely the 2d distribution that it affects, not just the ability to judge trend.

Reply:
Agreed, replacing with 

> making it hard to judge differences in the density of the data across the 2-dimonsional space as well as making it hard to spot the trend.